website_event_require_login/static/description/index.html

```diff
@@ -0,0 +1,83 @@
+<div class="container">
+            <div class="oe_styling_v8">
+				<section class="oe_container">
+					<div class="oe_row oe_spaced">
+						<h2 class="oe_slogan" style="color:#875A7B;">Tickets purchasing: force user to sign in / sign up</h2>
+						<h3 class="oe_slogan">User registration at your portal is always a plus for marketing</h3>
+					</div>
+				</section>
+
+				<section class="oe_container oe_dark">
+				  <div class="oe_row oe_spaced">
+					<div class="oe_span12">
+					  <div class="alert alert-warning" style="padding:4px;font-weight: 300; font-size: 20px;">
+						<p>
+						   The module updates /event/EVENT-ID/register page when user is not authenticated.
+						</p>
+					  </div>
+					</div>
+				  </div>
+				</section>
+
+				<section class="oe_container oe_dark">
+				  <div class="oe_row oe_spaced">
+					<h3 class="oe_slogan">How It Works</h3>
+					<h5 class="oe_slogan" style="font-size:16px;">
+								You can install this module in a <a title="How to install/update module" href="https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html">usual way</a>. After installation you need to configure.
+					</h5>
+					<div class="oe_span12 text-center">
+					  <p class="oe_mt32">
+						<font size="3px">Open page /event on Website as non-authenticated user and select some Event.<font>
+					  </font></font></p>
+					  <p class="oe_mt32">
+						<font size="3px">Button Register Now (Order Now if website_event_sale is installed) is renamed to ask for sign in / sign up.<font>
+					  </font></font></p>
+					  <p class="oe_mt32">
+						<font size="3px">As a result, if you click the button it opens login page sign in or sign up and then you are redirect back to event page and now you can order the tickets.<font>
+					  </font></font></p>
+					</div>
+					<div class="oe_row_img oe_centered">
+					  <img class="oe_demo oe_picture oe_screenshot" src="events-custom-fields-11.png">
```

Comment:
The screenshot is good, but some of the text are a bit technical and should be removed or sipmlified